sandbox/tests/timestamp-module_test.rs

```diff
@@ -0,0 +1,155 @@
+extern crate sandbox_execution_environment;
+use sc_executor::{WasmExecutor, WasmExecutionMethod};
+use sandbox_execution_environment::{Setup};
+use sp_core::{ traits::{ CallInWasm, Externalities, MissingHostFunctions}};
+use sp_runtime::{ traits::{BlakeTwo256 }};
+use sp_state_machine::TestExternalities as CoreTestExternalities;
+use parity_scale_codec::{Encode};
+use sp_wasm_interface::HostFunctions as _;
+pub use sp_inherents::{InherentData, InherentIdentifier, CheckInherentsResult, IsFatalError};
+type HostFunctions = sp_io::SubstrateHostFunctions;
+
+
+pub type TestExternalities = CoreTestExternalities<BlakeTwo256, u64>;
+
+fn call_in_wasm<E: Externalities> (
+    function: &str,
+    call_data: &[u8],
+    execution_method: WasmExecutionMethod,
+    ext: &mut E
+) -> Result<Vec<u8>, String> {
+    let setup = Setup::new();
+    let executor = crate::WasmExecutor::new(
+		execution_method,
+		Some(1024),
+		HostFunctions::host_functions(),
+		8,
+	);
+    executor.call_in_wasm(
+        &setup.wasm_code_array,
+        None,
+        function,
+        call_data,
+        ext,
+        MissingHostFunctions::Allow,
+    )
+}
+
+#[test]
+fn timestamp_set_works() {
+    let setup = Setup::new();
+    let mut ext = setup.ext;
+    let mut ext = ext.ext();
+
+    let mut now = vec![];
+    let mut did_update = vec![];
+
+    now.extend(b"timestamp".to_vec().encode());
+    did_update.extend(b"timestamp".to_vec().encode());
+    now.extend(b"now".to_vec().encode());
+    did_update.extend(b"didupdate".to_vec().encode());
+
+    let init_value: u64 = 42;
+    ext.set_storage(now.clone(), init_value.encode());
```

Comment:
why are you setting the timestamp here? My thinking is that the value should be set in the AS runtime right?

Reply:
I think setting the value in AS runtime and testing it again with get would introduce dependency to the `Timestamp.set()`. This approach is similar to the Rust implementation, but they have a special method in Timestamp `set_timestamp()` which is used only for testing.

Reply:
oh yeah you are right. You should set it using rust